pinax/stripe/models.py

```diff
@@ -37,6 +38,7 @@ class AccountRelatedStripeObject(StripeObject):
 
     class Meta:
         abstract = True
+        unique_together = ("stripe_id", "stripe_account")
```

Comment:
According to pinax maintainer, stripe treats every id as unique, even across accounts, in that case we shouldn't need this. and keeping `unique=True` is what we want.

Reply:
Still it doesn't apply on coupons. Coupon ids are set by the user.

Reply:
alright, then it looks to me this change should affect only coupons. not all models where stripe_id is guarantee to be unique globally.